src/core/shape_inference/include/fft_common_validation.hpp

```diff
@@ -62,13 +62,13 @@ void validate_axes(const ov::op::util::FFTBase* op,
                    std::vector<int64_t>& axes,
                    size_t input_rank,
```

Comment:
As future improvement consider change it to be int64_t as it is casted inside function. Maybe pass `Rank` object or for shape inference validation there will be required to pass all input shapes and rank can be check internally etc.

Reply:
Added as a part of https://github.com/openvinotoolkit/openvino/pull/19463.